lib/moon/assets/age_limit.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Moon.Assets.AgeLimit do
+  @moduledoc false
+
+  use Moon.StatelessComponent
+
+  import Moon.Helpers.Assets, only: [asset_name_to_filename: 1]
+
+  prop(name, :string)
+  prop(color, :string, values: Moon.colors())
+  prop(height, :string)
+  prop(width, :string)
+  prop(font_size, :string)
+  prop(vertical_align, :string)
+  prop(click, :event)
+  prop(class, :string)
+
+  def render(assigns) do
+    ~F"""
+    <svg
+      class={"moon-age_limit #{@class} #{@click && "cursor-pointer"}"}
+      :on-click={@click}
+      style={get_style(
+        color: @color,
```

Comment:
Would be it be possible to drop this style and replace this with Tailwind Style?

Reply:
I think this change can be done in another ticket.
We can do it under this new ticket https://coingaming.atlassian.net/browse/MDS-274
This one is for aligning with React and the tailwind version